packages/utilities/psammead-styles/package-lock.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "psammead-styles",
-  "version": "0.1.0",
+  "name": "@bbc/psammead-styles",
+  "version": "0.1.3",
```

Comment:
This seems to be jumping to `0.1.1` and `0.1.2`, could you explain this?

Reply:
We used 0.1.1 and 0.1.2 for non code changes https://github.com/BBC-News/psammead/blob/latest/packages/utilities/psammead-styles/CHANGELOG.md so we didnt publish